### DISCUSSION TITLE
Illustration placeholder size in the empty card component

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3479] [Help: Figma] [answered]
Hello, we're interested in replacing the illustrations in our empty state components with icons, as the current Paste illustrations don't align with the Segment style we're aiming for. We do have plans to introduce Segment-styled illustrations in the future.

We've encountered an issue in Figma because the current illustrations have fixed dimensions (192px). Is there a way to dynamically hug-content of the illustration placeholder slot so that we could put there avatar component and set dimensions in a variant? Thanks!

What we want to achieve:
<img width="1152" alt="Zrzut ekranu 2023-09-8 o 12 40 59" src="https://github.com/twilio-labs/paste/assets/140410476/3624f8b3-eef0-4aa3-95f4-a93c2af8ecc4">

Current limitation:
<img width="1148" alt="Zrzut ekranu 2023-09-8 o 12 49 20" src="https://github.com/twilio-labs/paste/assets/140410476/1930a6f5-6aee-4873-9563-52a183187eca">

**Comment by SiTaggart** [comment, marked as answer]
Hi there

Just adding a quick note about illustrations, as we don't actually own them. What we implement comes directly from Twilio Brand, which has recently launched a whole set of new illustration styles ready to be used in the product. We've started displaying and documenting them on the Docsite, first off under the [Empty State Pattern](https://paste.twilio.design/patterns/empty-state). We'll roll them out elsewhere over time.

The Brand team have worked very hard on the illustrations as part of the excellent brand refresh, and are expecting all Twilio products, including Segment, to use them. We also tend to agree and believe that only the Brand team should be leading the illustration style that we display in our Twilio product suite.

The Segment PD team should be reaching out to the Brand team to request any new illustrations if their use case isn't covered, but the current collection is vast and should be explored first. You might need a Segment PD representative to get into the brand asset collection and grab them for you until the Figma library is released, which we're told is any day now 🤞🏼. 

Using one of the approved new illustrations may make this issue redundant. Let us know how you get on.

**Reply by serifluous** [reply]
Re: Figma, It's been a known issue for us, unfortunately, because Figma gets finicky with sizing when swapping in components like this. Here's a workaround: https://forum.figma.com/t/instances-not-resizing-with-variable-swaps/1651/2

Just want to echo what Simon said, though, that the Brand team is putting together new illustrations that all of Twilio should be using.
